### PR TITLE
Enhancements/gauge update <?.x.x>

### PR DESCRIPTION
**Pull request by mattnolting**
- Increment threshold by 1
- Updates css and gauge stroke width
- Adds concept of negative or positive fill color

### CODE REVIEW

File: src/Charts/Gauge/Gauge.js [hunk 1]
```diff
@@ -59,18 +59,18 @@ class Gauge extends Component {
...
+            0: 'ins-m-fill-level-4',
+            1: 'ins-m-fill-level-3',
+            2: 'ins-m-fill-level-2',
+            3: 'ins-m-fill-level-1',
+            4: 'ins-m-fill-level-1'
         };
 
         const gaugeClasses = classNames(
             this.props.className,
+            'ins-m-' + this.props.gaugeFullCondition,
```
Comment (ryelo) [inline src/Charts/Gauge/Gauge.js]: Let's give this a default value

File: src/Charts/Gauge/Gauge.js [hunk 2]
```diff
@@ -97,7 +97,8 @@ Gauge.propTypes = {
     identifier: propTypes.string,
     label: propTypes.string,
     value: propTypes.number,
-    width: propTypes.number
+    width: propTypes.number,
+    gaugeFullCondition: propTypes.string
```
Comment (ryelo) [inline src/Charts/Gauge/Gauge.js]: Let's make this a bool instead of a string

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (karelhala) [inline .eslintrc]: This file shouldn't be here because we recently changed eslint rules.

File: package.json [hunk 4]
```diff
@@ -18,6 +18,9 @@
     "c3": "^0.6.7",
     "classnames": "^2.2.5",
     "d3": "^5.5.0",
+    "eslint-config-standard-jsx": "^6.0.2",
+    "eslint-loader": "^2.1.0",
+    "installing": "^1.0.0",
```
Comment (karelhala) [inline package.json]: I don't think we really need this package, do we?

File: src/Charts/Gauge/Gauge.js [hunk 5]
```diff
@@ -69,18 +69,20 @@ class Gauge extends Component {
...
-            3: 'ins-c-arc__low',
-            4: 'ins-c-arc__low'
+            0: 'ins-m-fill-level-4',
+            1: 'ins-m-fill-level-3',
+            2: 'ins-m-fill-level-2',
+            3: 'ins-m-fill-level-1',
+            4: 'ins-m-fill-level-1'
         };
 
+        let gaugeColor = this.props.flipFullColors ? "ins-m-negative" : "";
```
Comment (karelhala) [inline src/Charts/Gauge/Gauge.js]: Could be simplified and passed to `classNames`, he'll deal with it
```JS
const gaugeClasses = classNames(
  this.props.className,
  this.props.flipFullColors ? 'ins-m-negative',
  'ins-c-gauge',
  colors[Math.floor(this.props.value / threshold)]
)
```

File: src/PresentationalComponents/SimpleTableFilter/styles.scss [hunk 6]
```diff
@@ -1,2 +1,3 @@
+@import '~@patternfly/patternfly-next/sass-utilities/all';
 @import '~@patternfly/patternfly-next/components/InputGroup/styles.scss';
-@import '~@patternfly/patternfly-next/components/FormControls/styles.scss';
+@import '~@patternfly/patternfly-next/components/FormControl/styles.scss';
```
Comment (Kinlaw) [inline src/PresentationalComponents/SimpleTableFilter/styles.scss]: This should be
`@import '~@patternfly/patternfly-next/components/FormControls/styles.scss';`

File: src/Charts/Gauge/gauge.scss [hunk 7]
```diff
@@ -1,39 +1,72 @@
 @import '../../Utilities';
+@import '~@patternfly/patternfly-next/sass-utilities/all';
```
Comment (Kinlaw) [inline src/Charts/Gauge/gauge.scss]: This should be 
`@import '~@patternfly/patternfly-next/utilities/all';`

File: src/PresentationalComponents/SimpleTableFilter/styles.scss [hunk 8]
```diff
@@ -1,2 +1,3 @@
+@import '~@patternfly/patternfly-next/sass-utilities/all';
```
[the remark this answers is not in the document]
Comment (mattnolting) [reply src/PresentationalComponents/SimpleTableFilter/styles.scss]: The directory structure was updated a few releases ago
screen shot 2018-09-06 at 10 57 54 am

File: src/PresentationalComponents/SimpleTableFilter/styles.scss (hunk 6 above)
Comment (mattnolting) [reply src/PresentationalComponents/SimpleTableFilter/styles.scss]: ^^ was updated to `FormControl/`

**Pull request merged**